raiden-dapp/src/components/account/AccountContent.vue

```diff
@@ -139,10 +139,7 @@ export default class AccountContent extends Mixins(NavigationMixin) {
         },
       });
 
-      const mainAccount = this.$raiden.getMainAccount();
```

Comment:
what does this mean when `usingSubkey` is `undefined`? That we don't know or that we don't use subkeys?

Reply:
We agreed to put an assert here to express in code that at this point it is not expected (and theoretically impossible) that this property is undefined here.

Reply:
And roll back again. Our linter does not accept an assert, since this is part of the `console` namespace.
Anyway, Paul and I discussed this again. The subkey property gets set within the connect function of the RaidenService. This means whenever the app is connected, this property must be defined. So since the block we are discussing here is within a conditional block that checks if the app is connected, this statement will always evaluate a boolean value and it can't ever be undefined. At least as the code is at the moment. And do not see a reason why this should change, since the RaidenService can't know it if is using a sub-key or not until the connect function gets called.